### DISCUSSION TITLE
Time Bins/Video Increments

### DISCUSSION THREAD

**Discussion by maddyice03** [olivierfriard/BORIS#789] [Q&A]
Is there a way to have Boris automatically sort data according to time intervals within one video? I have a 10-minute video I want to analyze in 2-minute intervals. Can Boris sort the output based on which 2-minute interval the behavior occurred in? 

Alternatively, is there a way for the video to automatically pause every 2 minutes while viewing without having to create a new observation?

**Comment by olivierfriard** [comment]
Hi,
I do not understand very well your request. Please give me an example of your "interval sorted" output.

Otherwise you can use the scan sampling function as specified in the user guide:
https://www.boris.unito.it/user_guide/observations/#scan-sampling

**Reply by maddyice03** [reply]
We are interested in how the animal's behavior changes over time. For example, I'd like to be able to see for the first 2 mins, which behaviors occurred and how they compare to the behaviors in the last 2 mins.

Does scan sampling work with an observation created from a media file?
